packages/suite-web/src/static/index.html

```diff
@@ -34,6 +34,21 @@
                 script.setAttribute('src','//cdn.crowdin.com/jipt/jipt.js');
                 document.head.prepend(script);
             }
+
+            // switch favicon if dark mode
+            const icon = document.querySelector('link[rel=icon]')
+            const appleIcon = document.querySelector('link[rel=apple-touch-icon]')
+            const switchIcons = (isDM) => {
+              if (isDM) { // DM enabled
```

Comment:
use better variable name to avoid comment .e.g. `isDarkMode`

Reply:
7ceb865